SurfaceTopography/ChangeLog.md

```diff
@@ -36,6 +36,9 @@ v0.95.0 (not yet published)
 - ENH: Retrieving surface containers from https://contact.engineering
 - ENH: Added analysis functions for scale-dependent statistical properties of
   containers
+- ENH: Added `scanning_probe_reliability_cutoff` pipeline function to estimate
```

Comment:
We should also mentioned the API of the scale dependent derivatives changed (distance scaled by factor n)